frontend/src/index.tsx

```diff
@@ -107,7 +107,8 @@ const options: HighlightOptions = {
 	inlineStylesheet: true,
 	inlineImages: true,
 	sessionShortcut: 'alt+1,command+`,alt+esc',
-	version: import.meta.env.REACT_APP_COMMIT_SHA || undefined,
+	version: import.meta.env.REACT_APP_COMMIT_SHA ?? '1.0.0',
```

Comment:
Do we anticipate wanting this fallback to be updated, or more just a proof of concept that its working?